src/nifake/tests/test_session.py

```diff
@@ -328,3 +328,39 @@ def test_multiple_return_params(self):
             assert self.patched_errors.handle_error.call_count == 2
             self.patched_errors.handle_error.assert_called_with(session, self.patched_library.niFake_GetCalDateAndTime.return_value)
     '''
+
+    def test_get_vi_int32_attribute(self):
+        self.patched_library.niFake_GetAttributeViInt32.side_effect = self.side_effects_helper.niFake_GetAttributeViInt32
+        int = 3
+        self.side_effects_helper['GetAttributeViInt32']['attributeValue'] = int
+        with nifake.Session('dev1') as session:
+            attr_int = session.read_write_integer
+            assert(attr_int == int)
+            from mock import call
+            calls = [call(SESSION_NUM_FOR_TEST, b"", 1000004, ANY)]
+            self.patched_library.niFake_GetAttributeViInt32.assert_has_calls(calls)
```

Comment:
Why does this test use mock.call to check the call, while the next test uses assert_called_once_with()?